gazebo_color_plugin/src/color_plugin.cpp

```diff
@@ -0,0 +1,127 @@
+#include <gazebo/common/Plugin.hh>
+#include <ros/ros.h>
+#include <ros/console.h>
+#include "gazebo_color_sensor_plugin/color_sensor_plugin.h"
+
+#include "gazebo_plugins/gazebo_ros_camera.h"
+
+#include <string>
+#include <stdlib.h>
+
+#include <gazebo/sensors/Sensor.hh>
+#include <gazebo/sensors/CameraSensor.hh>
+#include <gazebo/sensors/SensorTypes.hh>
+#include <std_msgs/Bool.h>
+
+#include <sensor_msgs/Illuminance.h>
+
+#define COLOR_TOLERANCE 30
```

Comment:
Create these constants inside the class as `const` variables.

Reply:
Adressed on the PR referenced below